### PR TITLE
Fix for timezone adjusting

### LINKED ISSUE #1713: adjust-to-timezone code uses server local timezone, not the ARQ system default timezone
### Version

4.7.0

### What happened?

Test failure in TestFunctions2.adjust4 (new code for `ADJUST` in 4.8.0-SNAPSHOT).

Report https://lists.apache.org/thread/cvbs3qcbftztmtmco2dc5yxnk8lt51cb

### Relevant output and stacktrace

_No response_

### Are you interested in making a pull request?

Yes

### PR DESCRIPTION
**Pull request by afs**
GitHub issue resolved #1713

Fix for timezone adjusting and some clean-up.

----

 - [x] Tests are included.
 - [x] Commits have been squashed to remove intermediate development commit messages.
 - [x] Key commit messages start with the issue number (GH-xxxx or JENA-xxxx)

By submitting this pull request, I acknowledge that I am making a contribution to the Apache Software Foundation under the terms and conditions of the [Contributor's Agreement](https://www.apache.org/licenses/contributor-agreements.html).

### CODE REVIEW

File: jena-arq/src/main/java/org/apache/jena/sparql/expr/nodevalue/XSDFuncOp.java [hunk 1]
```diff
@@ -1171,12 +1168,12 @@ public static int compareDateTimeFO(NodeValue nv1, NodeValue nv2) {
         return x ;
     }
 
-    /** Return a normalized XMLGregorianCalendar appropriate for comparision */
+    /** Return a normalized XMLGregorianCalendar appropriate for F&O comparision using teh XSD algorithm. */
```
Comment (kinow) [inline jena-arq/src/main/java/org/apache/jena/sparql/expr/nodevalue/XSDFuncOp.java:1171]: s/comparision/comparison (not added in this PR, but worth fixing?). Also s/teh/the

File: jena-arq/src/main/java/org/apache/jena/sparql/function/StandardFunctions.java [hunk 2]
```diff
@@ -41,6 +41,13 @@
      * and sparql:* for all the SPARQL builtins.
      */
 
+    public static void loadOtherDefs(FunctionRegistry registry) {
+        // Only need to add functions that are not in org.apache.jena.sparql.function.library or have nemas that are not java-compliant
```
Comment (kinow) [inline jena-arq/src/main/java/org/apache/jena/sparql/function/StandardFunctions.java:45]: s/nemas/names

File: jena-arq/src/main/java/org/apache/jena/sparql/function/library/AFN_SystemTimezone.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    public AFN_SystemTimezone() {
+        super();
+    }
+
+    @Override
+    public NodeValue exec() {
+        return XSDFuncOp.localSystemTimezone();
+    }
+
+
```
Comment (kinow) [inline jena-arq/src/main/java/org/apache/jena/sparql/function/library/AFN_SystemTimezone.java:36]: Extra line, but doesn't matter.

**Review by kinow — APPROVED** [review]
Few typos, but +1! Thanks!

**Comment by afs** [comment]
All done - thanks!

**Pull request merged**